### ISSUE TITLE
two return value in class UnetBlock

### ISSUE THREAD

**Issue by Muzijiajian** [jantic/DeOldify#30]
As I check codes, I find that following codes in module.py file(https://github.com/jantic/DeOldify/blob/master/fasterai/modules.py)

def forward(self, up_p:int, x_p:int):
        up_p = self.tr_conv(up_p)
        x_p = self.x_conv(x_p)
        x = torch.cat([up_p,x_p], dim=1)
        x = self.relu(x)
        return self.out(x)
        return out

Why there is two return value in this function?

**Comment by jantic** [comment]
Wow yeah not sure how I wound up doing that.  Just a dumb mistake it appears. I'll remove it this afternoon.
